aten/src/ATen/core/jit_type.h

```diff
@@ -1061,7 +1061,7 @@ struct CAFFE2_API StringType : public Type {
     return rhs.kind() == kind();
   }
   std::string str() const override {
-    return "string";
+    return "str";
```

Comment:
This change seems unrelated. Can you split it into a different PR?

Reply:
It was a necessary fix to make the test in constant pooling work with this change. Previously "string" was being outputted and parsed as a typevar.